src/sentry/features/temporary.py

```diff
@@ -280,6 +280,10 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:performance-web-vitals-issues", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     # Enable Seer Suggestions for Web Vitals Module
     manager.add("organizations:performance-web-vitals-seer-suggestions", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    # Enable Web Vital issue ingest
```

Comment:
https://develop.sentry.dev/backend/issue-platform/#releasing-your-issue-type

i think we're missing the post process flag? i also think we want `api_expose` to be `False`?

Reply:
Apparently `GroupType.__init_subclass__` automatically registers the flags for unreleased types, so we actually don't need to declare them here. I've removed the changes to `temporary.py`!